testing/integration-testing-hooks.ts

```diff
@@ -1,17 +1,18 @@
-import semver from 'semver';
-// Installing @types/mongodb at the lerna root breaks some of the tests...
-// Yikes. Stick with ts-ignore for now.
-// @ts-ignore
-import {MongoClient, MongoClientOptions} from 'mongodb';
-import { promisify } from 'util';
 import child_process from 'child_process';
 import crypto from 'crypto';
+import { once } from 'events';
 import { promises as fs } from 'fs';
+import { Test } from 'mocha';
+// Installing @types/mongodb at the lerna root breaks some of the tests...
+// Yikes. Stick with ts-ignore for now.
+// @ts-ignore
```

Comment:
You can remove this, I think. With the 4.0 driver, we’re not using the `@types` package anymore anyway.